src/julia.h

```diff
@@ -232,6 +232,21 @@ typedef struct _jl_line_info_node_t {
     intptr_t inlined_at;
 } jl_line_info_node_t;
 
+typedef union {
+    struct {
+        uint8_t ipo_consistent  : 1;
+        uint8_t ipo_effect_free : 1;
+        uint8_t ipo_nothrow     : 1;
+        uint8_t ipo_terminates  : 1;
+        // Weaker form of `terminates` that asserts
+        // that any control flow syntactically in the method
+        // is guaranteed to terminate, but does not make
+        // assertions about any called functions.
+        uint8_t ipo_terminates_locally : 1;
+    } overrides;
+    uint8_t bits;
```

Comment:
accessing this will violate TBAA, do we care about making it correct?
```suggestion
    unsigned char bits;
```

Reply:
You mean union punning? IIRC this was legalized in C11.

Reply:
But regardless, I can comment out the bitfield, we don't actually access it from C at the moment (though we may want to in the future to set attributes for LLVM).

Reply:
IIRC, there is an exemption in the C11 standard if you use a literal `char` type, but not for type aliases that are equivalent to char.